### PR TITLE
Feature/242 participant nested filtering

### PR DESCRIPTION
**Pull request by celopacob**
Issue / Feature: https://github.com/cybsafe/phishtray/issues/242

### CODE REVIEW

File: participant/filters.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+class ExerciseListFilter(admin.SimpleListFilter):
+
+    title = _("Exercise")
+    parameter_name = "exercise__id__exact"
+
+    def lookups(self, request, model_admin):
+
+        exercises = (
+            Exercise.user_objects.filter_by_user(user=request.user)
+            .filter(trial_version=1)
```
Comment (fericsepi) [inline participant/filters.py]: Let's introduce a new method in the Exercise manager that would return only the records that belong to the org.
this filter_by_user will also return public exercises which should be excluded for non superadmin.

Comment (fericsepi) [reply participant/filters.py]: That method can also be used in #327  to produce the list of exercises that can go into the filter.

Comment (celopacob) [reply participant/filters.py]: You mean put this new method already on the filters?

Comment (celopacob) [reply participant/filters.py]: Done! Take a look please.

**Pull request merged**